roles/nginxplus/tasks/plus/setup-license.yml

```diff
@@ -1,16 +1,23 @@
 ---
 - name: "Create SSL Directory"
-  file:
+  ansible.builtin.file:
     path: /etc/ssl/nginx
     state: directory
-    mode: 0755
+    mode: "0755"
 
 - name: "Copy NGINX Plus Certificate and License Key"
-  copy:
+  ansible.builtin.copy:
     src: "{{ item }}"
     dest: /etc/ssl/nginx
     decrypt: true
-    mode: 0644
-  with_items:
+    mode: "0644"
+  loop:
     - "{{ nginx_license.certificate }}"
     - "{{ nginx_license.key }}"
+
+- name: "Copy NGINX Plus jwt"
+  ansible.builtin.copy:
+    src: "files/licence/nginx-repo.jwt"
+    dest: /etc/ssl/nginx/licence.jwt
```

Comment:
This looks like it follows the same pattern as the cert and license key above - same destination, same mode, also decrypted. Could we just add it to the loop in the previous task?

Reply:
If I am reading the [documentation correctly](https://docs.nginx.com/solutions/#jwt). Placing the file there will break it. See the additional information on "custom paths". On the other side of the upgrade process I think this will be OK. For now I like to stick to what the docs insist.

Reply:
Ah, it should be a different path, gotcha.